docker_kernel/kernel.py

```diff
@@ -156,25 +155,21 @@ def set_payload(self, source: str, text: str, replace: bool):
             "text": text,
             "replace": replace,
         }]
-                    
-    def tag_image(self, image_id: str, name: str, tag: str|None=None):
+
+    def tag_image(self, image: str, name: str, tag: str|None=None):
         """ Tag an image.
         Parameters
         ----------
-        image_id: str
-            Id of image to be saved.
+        image: str
+            Id or name of the image to be tagged.
         name: str
             Image name to be assigned.
         tag: str, optional
             Typically a specific version or variant of an image.
-        
+
         Return
         ------
         None
         """
-        tag = self.default_tag if tag is None else tag
-
-        if name not in self._tags:
-            self._tags[name] = {}
 
-        self._tags[name][tag] = image_id
+        docker.client.from_env().images.get(image).tag(name,tag)
```

Comment:
Lets just use `self._api.tag(image, name, tag)` seems a little less messy. Also with this we don't need to specify "latest" as our default in the `tag` magic (When `self._api.tag` is provided `None` as its `tag` it will use dockers default -> latest)

Reply:
Good point!
Done